api/src/main/java/org/openmrs/api/ProgramWorkflowService.java

```diff
@@ -530,4 +535,36 @@ public void purgeConceptStateConversion(ConceptStateConversion conceptStateConve
 	 * @should throw an error when multiple program states with same uuid are found
 	 */
 	public ConceptStateConversion getConceptStateConversionByUuid(String uuid);
+
```

Comment:
@brayoh Add public modifier to all the methods in this class

Reply:
I've added a public modifiers for all the methods

Reply:
I've added a public modifiers for all the methods